static/app/views/performance/transactionSummary/transactionReplays/index.tsx

```diff
@@ -114,6 +119,22 @@ function generateEventView({
   location: Location;
   transactionName: string;
 }) {
+  const fields = [
+    'replayId',
+    'count()',
+    SPAN_OP_RELATIVE_BREAKDOWN_FIELD,
+    'transaction.duration',
+    'trace',
+    'timestamp',
+  ];
+
+  const breakdown = decodeFilterFromLocation(location);
+  if (breakdown !== SpanOperationBreakdownFilter.None) {
+    fields.splice(2, 1, `spans.${breakdown}`);
+  } else {
+    fields.push(...SPAN_OP_BREAKDOWN_FIELDS);
+  }
+
```

Comment:
This might be easier to read & modify in the future if we just wrote the arrays that we want to see. I'm not sure why the splice has to be in the middle, but it's adding some mystery i don't want to sort out.

```javascript
if (...) {
  fields = [
    'replayId',
    'count()',
    'transaction.duration',
    'trace',
    'timestamp',
   `spans.${breakdown}`,
  ]
} else {
  fields = [
'replayId',
    'count()',
    SPAN_OP_RELATIVE_BREAKDOWN_FIELD,
    'transaction.duration',
    'trace',
    'timestamp',
    ...SPAN_OP_BREAKDOWN_FIELDS
  ];
}
```

and now that i've sorted the fields, a couple push calls would be enough and the base fields get shared:
```
const fields = [
  'replayId',
  'count()',
  'transaction.duration',
  'trace',
  'timestamp',
];
if (...) {
  fields.push(`spans.${breakdown}`)
} else {
  fields.push(SPAN_OP_RELATIVE_BREAKDOWN_FIELD)
  fields.push(...SPAN_OP_BREAKDOWN_FIELDS)
}
```